bpf/tests/tc_nodeport_l3_dev.c

```diff
@@ -0,0 +1,374 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+/* Copyright Authors of Cilium */
+
+#include "common.h"
+
+#include <bpf/ctx/skb.h>
+#include <bpf/helpers_skb.h>
+#include "pktgen.h"
+
+#define ETH_HLEN		0
+#define SECCTX_FROM_IPCACHE	1
+#define ENABLE_HOST_ROUTING
+#define ENABLE_IPV4
+#define ENABLE_IPV6
+#define SKIP_ICMPV6_NS_HANDLING
+
+#define TEST_IP_LOCAL		v4_pod_one
+#define TEST_IP_REMOTE		v4_pod_two
+#define TEST_IPV6_LOCAL		v6_pod_one
+#define TEST_IPV6_REMOTE	v6_pod_two
+#define TEST_LXC_ID_LOCAL	233
+
+/* We wanted to tail call handle_policy from bpf_lxc, but at present it's
+ * impossible to #include both bpf_host.c and bpf_lxc.c at the same time.
+ * Therefore, we created a stud, mock_hanle_policy, to simply check if the
+ * our skb reaches there.
```

Comment:
Not worth a respin, but if you need to update your PR for another reason:

```suggestion
 * Therefore, we created a stud, mock_hanle_policy, to simply check if
 * our skb reaches there.
```

There's another typo in the description of your first commit: `lagacy`.